Quickref.md

```diff
@@ -48,17 +48,22 @@ Since MonadHold depends on MonadSample, any [S] function also runs in [H] contex
 [ ]   attachPromptlyDynWithMaybe :: (a -> b -> Maybe c) ->  Dynamic a -> Event b -> Event c
 
 -- Combine multiple Events
-[ ]   <>        :: Monoid a => Event a -> Event a -> Event a
+[ ]   <>        ::         Monoid a => Event a -> Event a -> Event a
```

Comment:
This actually has a `Semigroup a` context, not `Monoid a`

https://github.com/reflex-frp/reflex/blob/develop/src/Reflex/Class.hs#L574